### ISSUE TITLE
Write search querying integrated tests

### ISSUE THREAD

**Issue by seanh** [hypothesis/product-backlog#643]
Actually write all of the tests needed for the existing search querying code.

See also this separate card [Define integrated test approach for existing search querying code](https://github.com/hypothesis/product-backlog/issues/639) that will define _how_ these tests are going to be designed.

Once #639 is done we can add a checklist to this card for what needs to be tested, and perhaps break it down into separate cards.

**Comment by seanh** [comment]
This got broken up into smaller cards, see https://github.com/hypothesis/product-backlog/issues/639#issuecomment-392480090
